jena-core/src/main/java/org/apache/jena/reasoner/rulesys/Rule.java

```diff
@@ -1080,6 +1092,7 @@ private Rule doParseRule(boolean retainVarMap) {
                 r.isBackward = backwardRule;
                 return r;
             } catch (NoSuchElementException e) {
+                e.printStackTrace();
```

Comment:
Debug?

Reply:
Arg! I've put that in, then taken it out, then put it in ...
Thanks for spotting that.